ci/ansible/roles/pulp-backup/defaults/main.yaml

```diff
@@ -0,0 +1,20 @@
+---
+# location of backup on remote
+fetch_to_localhost: true
+remote_bkup_dir: "{{ ansible_user_dir }}/remote_backups"
+local_bkup_dir: "local_backups"
```

Comment:
The `pulp-backup` and `pulp-restore` roles have different default values for the `local_bkup_dir` variable. This needs to be fixed.